aleph_scoring/__main__.py

```diff
@@ -335,11 +335,18 @@ def export_as_html(input_file: Optional[Path]):
     os.system("jupyter nbconvert --execute Node\\ Score\\ Analysis.ipynb --to html")
 
 
-def main():
+def get_sentry_dsn(settings_):
```

Comment:
the parameter is never used because it has a "_" at the end of its name, is it wanted?

Reply:
Fixed, renamed as `app_settings`